src/controllers/accommodation.js

```diff
@@ -340,15 +341,15 @@ class accommodation {
       } = req;
       const pagination = paginate(page, limit);
       const allAccommodations = await Accommodations.findAll({
-        attributes: {
-          exclude: ['createdBy', 'imageUrl', 'country', 'createdAt', 'updatedAt']
-        },
         include: [{
           model: Rooms,
           attributes: {
             exclude: ['accommodationsID', 'createdBy', 'createdAt', 'updatedAt']
           }
         }],
+        include: [{
+          model: AcommodationLikesAndUnlikes,
```

Comment:
I would advise you to add comments too made on accommodation.
so that since you will be having all accommodation in redux when you are trying to display one accommodation you won't have to to make a call to the backend too.

Reply:
yeah that will be better

Reply:
@blaise82 just a quite short correction, there are feedbacks instead of comments. Thanks, I added that ka array as well.